j2ee.sun.ddui/licenseinfo.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/j2ee/sun/share/plan/deployment-plan.dtd</file>
```

Comment:
I suppose that this is used here:
https://github.com/apache/incubator-netbeans/blob/91d94516c76ab50e8b420f70633694c025ca516e/j2ee.sun.ddui/src/org/netbeans/modules/j2ee/sun/share/plan/DeploymentPlan.java#L87-L101

Reply:
The DTD is the basis and DeploymentPlan is generated from it. So the IMHO correct solution would be to add the ALv2 header to the DTD file. The file is not a template, is not displayed, so should be treated as all other source files.